ssg/build_remediations.py

```diff
@@ -966,18 +871,17 @@ def expand_xccdf_subs(fix, remediation_type, remediation_functions):
         modfixtext = "".join(modfix)
         # Don't perform sanity check at bash comments because they are not substituted
         modfixtext = re.sub(r'#.*', '', modfixtext)
-        for func in remediation_functions:
-            # Then efine expected XCCDF sub element form for this function
-            funcxccdfsub = "<sub idref=\"function_%s\"" % func
-            # Finally perform the sanity check -- if function was properly XCCDF
-            # substituted both the original function call and XCCDF <sub> element
-            # for that function need to be present in the modified text of the fix
-            # Otherwise something went wrong, thus exit with failure
-            if func in modfixtext and funcxccdfsub not in modfixtext:
-                sys.stderr.write("Error performing XCCDF <sub> substitution "
-                                 "for function %s in %s fix. Exiting...\n"
-                                 % (func, fix.get("rule")))
-                sys.exit(1)
+        # Then efine expected XCCDF sub element form for this function
+        funcxccdfsub = "<sub idref=\"function_%s\"" % "bash-populate"
```

Comment:
We probably don't want to have any element with ID starting with `function` anywhere, we want to get rid of them.